core/src/sigverify_stage.rs

```diff
@@ -254,27 +300,40 @@ impl SigVerifyStage {
         let mut last_print = Instant::now();
         Builder::new()
             .name("solana-verifier".to_string())
-            .spawn(move || loop {
-                if let Err(e) =
-                    Self::verifier(&packet_receiver, &verified_sender, &verifier, &mut stats)
-                {
-                    match e {
-                        SigVerifyServiceError::Streamer(StreamerError::RecvTimeout(
-                            RecvTimeoutError::Disconnected,
-                        )) => break,
-                        SigVerifyServiceError::Streamer(StreamerError::RecvTimeout(
-                            RecvTimeoutError::Timeout,
-                        )) => (),
-                        SigVerifyServiceError::Send(_) => {
-                            break;
+            .spawn(move || {
+                let mut bloom = Bloom::random(1_000_000, 0.0001, 1 << 20).into();
```

Comment:
Limiting the size to 2^20 bits (131KB) is equivalent to accepting a 60% failure rate for a size of 1M items

```
    #[test]
    fn check_failure_rate() {
        let b: Bloom<Hash> = Bloom::random(1_000_000, 0.60, usize::MAX);
        assert_eq!(b.bits.len(), 1_063_216); // ~= 2^20 (1,048,576)
    }
```

Reply:
ah thanks!  I thought it was bytes.  8<<20 is what I meant, which is 1/20k failure rate.